ecommerce/frontend/src/components/product/ProductExtra.less

```diff
@@ -0,0 +1,26 @@
+@import '../../common.less';
+
+.product-extra {
+    padding: 16px;
+    background: #f0f2f5;
+    .product-extra-price {
+        font-size: 2.5em;
+        .product-extra-old-price {
+            font-weight: lighter;
+            text-decoration: line-through;
```

Comment:
I think, the discounted price is hard to read. Black line on top of black font color. We can try out new colors. 

`text-decoration: line-through #E2BE5A; `  is better but not the best

Reply:
I'll try that one, thanks